srcpkgs/singular/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'singular'
+pkgname=singular
+version=4.2.1
+revision=1
+_own_version=4-2-1
```

Comment:
Could you use something like `${version//./-}`? Also remove this variable, because it's used only once.